app/models/picture.rb

```diff
@@ -0,0 +1,31 @@
+class Picture < ActiveRecord::Base
+
+  belongs_to :user
+  has_many :comments, dependent: :destroy
+  has_many :likes, dependent: :destroy
+  validates :title, :image, presence: true
+
+  has_attached_file :image,
+                    :styles => {  :medium => "800x800>",
+                                  :thumb => "200x200>" },
+                    :default_url => "/images/:style/missing.jpeg"
+  validates_attachment_content_type :image, :content_type => /\Aimage\/.*\Z/
+
+  def was_created_by?(user)
+    user.id === self.user_id
```

Comment:
Style/CaseEquality: Avoid the use of the case equality operator `===`. (https://github.com/bbatsov/ruby-style-guide#no-case-equality)